recipe/meta.yaml

```
@@ -13,7 +12,7 @@ package:
 
 source:
   url: https://github.com/AOMediaCodec/{{ name }}/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 914662e16245e062ed73f90112fbb4548241300843a7772d8d441bb6859de45b
+  sha256: 0a545e953cc049bf5bcf4ee467306a2f113a75110edf59e61248873101cd26c1
 
 build:
   number: 1
```

Change of version -> build_number 0?

```suggestion
  number: 0
```